### PR TITLE
WW-314 fix them buttonz

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

**Comment by sebastianmarzjan** [comment]
LGTM, though now we have the 6px spacing defined in at least two places of community page. Can you unify it?

**Comment by dianafa** [comment]
14px spacing makes much more sense. LGTM

File: extensions/wikia/CommunityPage/styles/components/_admins-module.scss [hunk 1]
```diff
@@ -18,23 +16,25 @@ $admins-view-all-padding: 18px;
...
 
 	.community-page-admins-view-all {
 		color: $color-buttons;
-		display: block;
+		display: inline-block;
 		font-size: $typescale-size-minus-2;
 		font-weight: bold;
-		padding: $admins-view-all-padding 0;
+		line-height: $line-height;
+		margin: $content-spacing-xsmall 0 $content-spacing-xxsmall 0;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_admins-module.scss]: You don't need last `0`

Comment (sebastianmarzjan) [reply extensions/wikia/CommunityPage/styles/components/_admins-module.scss]: Actually, you could go with 

```
margin: $content-spacing-xsmall 0;
```

in this case

Comment (idradm) [reply extensions/wikia/CommunityPage/styles/components/_admins-module.scss]: Nope, I couldn't. They have a different number of `x`s in them. ;)

Comment (sebastianmarzjan) [reply extensions/wikia/CommunityPage/styles/components/_admins-module.scss]: Oh, the double x'es, how easy they are to miss 🙈

**Comment by lukaszkonieczny** [comment]
LGTM

**Pull request merged**